code/datums/jobs.dm

```diff
@@ -1241,11 +1243,13 @@ ABSTRACT_TYPE(/datum/job/civilian)
 /datum/job/special/medical_assistant
 	name = "Medical Assistant"
 	linkcolor = "#9900FF"
-	limit = 1
+	limit = 2
 	wages = PAY_UNTRAINED
 	low_priority_job = 1
-	slot_jump = list(/obj/item/clothing/under/color/white)
+	slot_jump = list(/obj/item/clothing/under/scrub,/obj/item/clothing/under/scrub/teal,/obj/item/clothing/under/scrub/blue,/obj/item/clothing/under/scrub/purple,/obj/item/clothing/under/scrub/orange,/obj/item/clothing/under/scrub/pink)
 	slot_foot = list(/obj/item/clothing/shoes/white)
+	slot_ears = list(/obj/item/device/radio/headset/medical)
+	slot_belt = list(/obj/item/device/pda2/medical) //a special PDA since medical gets a lot of overwhelming alerts
```

Comment:
I don't quite get how is this special. Isn't that just the path of the regular medical PDA?

Reply:
I originally had made a special med-ass PDA that had no port-a-med or nano-med in order to prevent griefy tactics, but after some consideration on how not having those could harm the learning experience, and how a med cart is 160 credits to summon the nano-med anywhere, I decided to cut it.